cloudformation/membership-attribute-service.yaml

```diff
@@ -0,0 +1,329 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Membership Attributes service
+Parameters:
+  Name:
+    Description: The name given to the autoscaling group
+    Type: String
+  KeyName:
+    Description: The EC2 Key Pair to allow SSH access to the instances
+    Type: AWS::EC2::KeyPair::KeyName
+  SiteDomain:
+    Description: Site domain Name
+    Type: String
+    Default: members-data-api.theguardian.com
+  Stage:
+    Description: Environment name
+    Type: String
+    Default: PROD
+  InstanceType:
+    Description: EC2 instance type
+    Type: String
+    Default: t2.small
+    AllowedValues:
+    - t2.micro
+    - t2.small
+    - t2.medium
+    - m3.medium
+    - m3.large
+    - m3.xlarge
+    - m3.2xlarge
+    - c3.large
+    - c3.xlarge
+    - c3.2xlarge
```

Comment:
I know it's not been introduced in this PR, but we can probably trim this list down a bit a some stage.